src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+| Pointer Location | Pressing the `Ctrl` key creates a visual notification at the mouse pointer location. |
+| Middle Click Paste | Clicking the middle mouse button pastes content from the clipboard. |
+| (Touchpad) Disable While Typing | The touchpad is disabled when using the keyboard. |
+| (Touchpad) Mouse Click Emulation | Select touchpad input that will be interpreted as input from a standard mouse: <ul><li>Fingers: Click with two fingers on the touch pad to input right-click, and three fingers input a middle-click.</li><li>Area: Click the bottom right of the touch pad to input right-click, and the bottom middle for a middle-click.</li><li>Disabled: The touch pad input will not be interpreted as right, middle, or left-click actions. |
+
+### Startup Applications
+
+Choose applications that will automatically start after you log in.
+
+![Startup App Tweaks](/images/gnome-tweaks-extensions/startup-app-tweaks.png)
+
+Add an application by clicking the `+`, selecting an application from the list, and then clicking `Add`.
+
+![Select Startup App](/images/gnome-tweaks-extensions/select-startup-app.png)
+
+### Top Bar
+
+`Top Bar` settings allow you to configure the clock to display additional information.
+
+![Top Bar Tweaks](/images/gnome-tweaks-extensions/top-bar-tweaks.png)
+
+You can also choose to display week numbers in the system calendar application.
+
+![Calendar Tweaks](/images/gnome-tweaks-extensions/calendar-tweaks.png)
+
+### Window Titlebars
+
+These settings allow you to interact with windows using mouse clicks. You can also choose available titlebar buttons and their placement. Right, middle, and left-click buttons can each be configured to perform one of the following actions when clicking a window's titlebar:
+
+![Titlebar Tweaks](/images/gnome-tweaks-extensions/titlebar-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Lower | A "send to back" action occurs. |
+| Menu | Display a window manipulation menu. |
+| Minimize | Minimize a window.|
+| Toggle Maximize | Maximize a window, the window returns to its previous size on second click. |
+| Toggle Maximize Horizontally* | Extend a window to the left and right sides of the screen without modifying vertical dimensions. |
+| Toggle Maximize Vertically* | Expand a window to the top and bottom of the screen without modifying horizontal dimensions. |
+| Toggle Shade* | Collapse or expand a window from its titlebar when the window's titlebar is clicked. |
+
+\* This feature is not supported by all application windows.
+
+### Windows
+
+The `Windows` menu offers additional features for interacting with system and application windows.
+
+![Windows Tweaks](/images/gnome-tweaks-extensions/windows-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Attach Modal Dialogs | Modal dialogs (such as windows that spawn when performing a "Save as...") stay in a fixed position within their parent window. |
```

Comment:
While this description is accurate, I think the more important implication is that the dialogue *can't* be moved independently from the parent window, and moving the dialogue will also move the parent window.